overlord/snapstate/snapstate_test.go

```diff
@@ -3523,9 +3532,6 @@ SNAPD_APPARMOR_REEXEC=1
 
 	st := s.state
 	st.Lock()
-	// ensure that only this specific snap is installed
```

Comment:
I'm not sure what the intent of this was, so removing it may be wrong.

Reply:
this can and should be undone now